systems/plants/ContactForceTorqueSensor.m

```diff
@@ -119,7 +119,7 @@
       % contact normal and tangential directions in world coordinates
       nD = length(tsmanip.LCP_cache.data.contact_data.d);
       nC_body = nA + nB;
-      nC = manip.getNumContactPairs;
+      nC = size(tsmanip.LCP_cache.data.contact_data.normal,2);
```

Comment:
because it will be a (changing) subset of all possible contact pairs now.  i've added a comment